src/witan/app/handler.clj

```diff
@@ -23,7 +23,7 @@
 
 (def LoginReturn
   (s/either {(s/required-key :token) s/Str
-             (s/required-key :id) s/Str}
+             (s/required-key :id) s/Uuid}
```

Comment:
No, cassandra returns a `java.util.UUID` which wasn't being cast to a string so the schema on LoginReturn was failing. Personally, I prefer the Uuid check but as an alternative to this fix, would you prefer me to cast the Uuid to a string before returning?